modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -75,26 +211,26 @@ include::partial$include-nav-organizations.adoc[]
   +
```

Comment:
This and the one above aren't formatting correctly in the build.

Reply:
@hannanelson They all look fine in my local build. What are you seeing?